aws/resource_aws_elasticache_cluster_test.go

```diff
@@ -1208,168 +1220,192 @@ resource "aws_security_group" "bar" {
-  node_type         = "cache.m3.medium"
-  num_cache_nodes   = 1
-  port              = 6379
+resource "aws_elasticache_cluster" "test" {
+    apply_immediately = true
+    cluster_id        = "%[1]s"
+    engine            = "redis"
+    engine_version    = "%[2]s"
+    node_type         = "cache.m3.medium"
+    num_cache_nodes   = 1
+    port              = 6379
 }
 `, rName, engineVersion)
 }
 
 func testAccAWSElasticacheClusterConfig_NodeType_Memcached_Ec2Classic(rName, nodeType string) string {
 	return fmt.Sprintf(`
-resource "aws_elasticache_cluster" "bar" {
-  apply_immediately = true
-  cluster_id        = "%[1]s"
-  engine            = "memcached"
-  node_type         = "%[2]s"
-  num_cache_nodes   = 1
-  port              = 11211
+resource "aws_elasticache_cluster" "test" {
+    apply_immediately = true
+    cluster_id        = "%[1]s"
+    engine            = "memcached"
+    node_type         = "%[2]s"
+    num_cache_nodes   = 1
+    port              = 11211
 }
 `, rName, nodeType)
 }
 
 func testAccAWSElasticacheClusterConfig_NodeType_Redis_Ec2Classic(rName, nodeType string) string {
 	return fmt.Sprintf(`
-resource "aws_elasticache_cluster" "bar" {
-  apply_immediately = true
-  cluster_id        = "%[1]s"
-  engine            = "redis"
-  node_type         = "%[2]s"
-  num_cache_nodes   = 1
-  port              = 6379
+resource "aws_elasticache_cluster" "test" {
+    apply_immediately = true
+    cluster_id        = "%[1]s"
+    engine            = "redis"
+    node_type         = "%[2]s"
+    num_cache_nodes   = 1
+    port              = 6379
 }
 `, rName, nodeType)
 }
 
 func testAccAWSElasticacheClusterConfig_NumCacheNodes_Redis_Ec2Classic(rName string, numCacheNodes int) string {
 	return fmt.Sprintf(`
-resource "aws_elasticache_cluster" "bar" {
-  apply_immediately = true
-  cluster_id        = "%[1]s"
-  engine            = "redis"
-  node_type         = "cache.m3.medium"
-  num_cache_nodes   = %[2]d
-  port              = 6379
+resource "aws_elasticache_cluster" "test" {
+    apply_immediately = true
+    cluster_id        = "%[1]s"
+    engine            = "redis"
+    node_type         = "cache.m3.medium"
+    num_cache_nodes   = %[2]d
+    port              = 6379
 }
 `, rName, numCacheNodes)
 }
 
 func testAccAWSElasticacheClusterConfig_ReplicationGroupID_InvalidAttribute(rName, attrName, attrValue string) string {
 	return fmt.Sprintf(`
-resource "aws_elasticache_cluster" "replica" {
-  cluster_id           = "%[1]s"
-  replication_group_id = "non-existent-id"
-  %[2]s                = "%[3]s"
+resource "aws_elasticache_cluster" "test" {
+    cluster_id           = "%[1]s"
+    replication_group_id = "non-existent-id"
+    %[2]s                = "%[3]s"
 }
 `, rName, attrName, attrValue)
 }
 
 func testAccAWSElasticacheClusterConfig_ReplicationGroupID_AvailabilityZone_Ec2Classic(rName string) string {
 	return fmt.Sprintf(`
-data "aws_availability_zones" "available" {}
+data "aws_availability_zones" "test" {}
```

Comment:
This one can actually stay "available" -- that's consistent with other testing.

Reply:
👍